testing/himem_test/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config TESTING_HIMEM_TEST
+	tristate "HIMEM TEST APP"
+	---help---
+		Enable the SMP APP
+
+if TESTING_HIMEM_TEST
+
+config TESTING_HIMEM_TEST_PROGNAME
+	string "Program name"
+	default "himem_test"
+	---help---
+		This is the name of the program that will be used when the NSH ELF
+		program is installed.
```

Comment:
NSH ELF program?

Reply:
This is a generic description in Kconfig.